tensorflow/tools/docker/parameterized_docker_build.sh

```diff
@@ -87,6 +87,10 @@
 #   TF_DOCKER_BUILD_OPTIONS
 #     (Optional)
 #     Specifies the desired build options. Defaults to OPT.
+#
+#   TF_BAZEL_BUILD_OPTIONS
```

Comment:
you've introduced this env (and mechanism for it to work), but I can't see any place where you actually use it? Why?

Reply:
See line 332.

Reply:
yeah you allowed to set TF_BAZEL_BUILD_OPTIONS and provided mechanism for it to work. I assume what will actually use this mechanism is part of CI somewhere? Not in this change?

Reply:
Correct. We have scripts internally that use it. tensorflow/tools/ci_build/linux/cpu/run_mkl.sh could (and probably should) use it. Right now, that script just takes default compiler x86 optimizations. tensorflow/tools/ci_build/ci_parameterized_build.sh has a TF_BUILD_OPTIONS parameter that is close, but doesn't give our scripts the flexibility they need.

Reply:
OK, makes sense. Normally I'd say we should do it in separate PR as this is different feature/problem, but that's ok. Smaller PRs are just easier to review